shared/chat/conversation/edit-popup.desktop.js

```diff
@@ -74,8 +69,18 @@ class EditPopup extends Component<void, Props, void> {
             width: messageRect.width,
           }} />
         <Box style={{...globalStyles.flexBoxRow, justifyContent: 'center'}}>
-          <Button type='Secondary' label='Cancel' onClick={() => { setImmediate(onClose) }} />
-          <Button type='Primary' label='Save' onClick={() => { setImmediate(this._onSave) }} />
+          <Button type='Secondary' label='Cancel' onClick={e => {
+            e.preventDefault()
+            setImmediate(onClose)
+          }} />
+          <Button type='Primary' label='Save' onClick={e => {
+            e.preventDefault()
+            const value = this._input && this._input.getValue()
```

Comment:
@chromakode i had to do this in order to get the save working. very tricky. if the old setImmediate was there at that point _input would be set back to null since it'd been unmounted by the time setImmediate happens. Really looking forward to the portal library :)